content/en/actions/app_builder/embedded_apps/_index.md

```diff
@@ -4,6 +4,7 @@ description: Embed published apps in dashboards and sync them with template vari
 disable_toc: false
 aliases:
     - /service_management/app_builder/embedded_apps
+    - /actions/app_builder/embedded_apps/
```

Comment:
```suggestion
```
This redirect is causing that job failure. `embedded_apps/_index.md` == `/embedded_apps/`